packages/mobile/src/screens/chat-screen/ChatMessagePlaylist.tsx

```diff
@@ -0,0 +1,158 @@
+import { useCallback, useMemo } from 'react'
+
+import type { ChatMessageTileProps, ID, TrackPlayback } from '@audius/common'
+import {
+  Kind,
+  PlaybackSource,
+  QueueSource,
+  accountSelectors,
+  getPathFromPlaylistUrl,
+  makeUid,
+  playerSelectors,
+  useGetPlaylistById,
+  useGetTracksByIds,
+  usePlayTrack,
+  usePauseTrack,
+  parsePlaylistIdFromPermalink
+} from '@audius/common'
+import { useSelector } from 'react-redux'
+
+import { CollectionTile } from 'app/components/lineup-tile'
+import { make, track as trackEvent } from 'app/services/analytics'
+
+const { getUserId } = accountSelectors
+const { getUid, getPlaying, getTrackId } = playerSelectors
+
+export const ChatMessagePlaylist = ({
+  link,
+  onEmpty,
+  onSuccess,
+  styles
+}: ChatMessageTileProps) => {
+  const currentUserId = useSelector(getUserId)
+  const isPlaying = useSelector(getPlaying)
+  const playingTrackId = useSelector(getTrackId)
+  const playingUid = useSelector(getUid)
+
+  const playlistId = parsePlaylistIdFromPermalink(
+    getPathFromPlaylistUrl(link) ?? ''
+  )
+  const { data: playlist } = useGetPlaylistById(
+    {
+      playlistId,
+      currentUserId
+    },
+    { disabled: !playlistId }
+  )
+  const collection = playlist
```

Comment:
This will make `collection` change on every render even when the playlist doesn't. Just making sure that's alright.